### PR TITLE
Fixes on italian translation

### PR DESCRIPTION
**Pull request by EdoardoTona**

### CODE REVIEW

**Comment by SebastianStehle** [comment]
Please read this guide: https://docs.squidex.io/01-getting-started/contributing-and-developing/translating

File: backend/i18n/frontend_it.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "api.contentApi": "Content API",
```
Comment (SebastianStehle) [inline backend/i18n/frontend_it.json:2]: Please check the docs

**Review by SebastianStehle — CHANGES_REQUESTED** [review]

File: backend/i18n/frontend_it.json (hunk 1 above)
Comment (EdoardoTona) [reply backend/i18n/frontend_it.json:2]: ah sorry, I have to edit the source jsons

**Comment by SebastianStehle** [comment]
Thank you very much ❤️

**Pull request merged**